### ISSUE TITLE
Training Details for CDM

### ISSUE THREAD

**Issue by wenzhoulyu** [ByteDance-Seed/manip-as-in-sim-suite#1]
Excellent work! I was reviewing the paper and the associated code for CDM and was wondering if you could provide more details about the training strategy. Specifically, I would appreciate it if you could share the learning rate, batch size, number of epochs, and any other key hyperparameters used during training. This information would be extremely helpful for reproducing the results presented in the paper.

**Comment by Ericonaldo** [comment]
Hi! Thanks for liking this work!

Here are some training parameters you mentioned; they are just normal. The key is the guided filter to bake the simulation training data.

max_epochs: 11
devices: 8
num_nodes: 8

train_loader_opts:
    batch_size: 1

optimizer:
    _target_: torch.optim.AdamW
    lr: 0.00005
    weight_decay: 0.01

**Comment by wenzhoulyu** [comment]
Thank you so much!
